website/docs/r/service_template.html.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: "opennebula"
+page_title: "OpenNebula: opennebula_service_template"
+sidebar_current: "docs-opennebula-resource-service-template"
+description: |-
+  Provides an OpenNebula service template resource.
+---
+
+# opennebula_service_template
+
+Provides an OpenNebula service template resource.
+
+This resource allows you to manage service templates on your OpenNebula clusters. When applied,
+a new service template will be created. When destroyed, that service template will be removed.
+
+## Example Usage
+
+```hcl
+resource "opennebula_service_template" "stemplate" {
+    name = "tf-stemplate"
+    template = templatefile("${path.module}/template.json", {})
```

Comment:
this syntax is not correct now since terraform 0.12 release. Can you update it ?

Reply:
Sure, I'll look into that.